src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,52 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { Tab } from '../../interfaces/Tab';
+
+interface Props {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelect: (tabId: string) => void;
+}
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelect,
+}) => {
+  const shouldUpdate = (tabId: string) => {
+    let selectedId = tabId;
+
+    if (!tabs.map(tab => tab.id).includes(selectedId)) {
+      selectedId = tabs[0].id;
+    }
```

Comment:
You could decide should we update or not while rendering `tabs` below

Reply:
oh. may be I got that. May be you mean I had to put whatever id into `state `and validate displayed tab in render method.
But I think it is better to keep valid `id`'s in `state` than reacting to errors whenever we want to use `selectedTabId`.